### PR TITLE
Style / contribution guides, and typo fix

### PR DESCRIPTION
**Pull request by SueSmith**
Adds docs style guide and reinstates contributing guide since it disappeared and link in readme is broken (have also linked code of conduct to forum one as the repo one was just linking there anyway), also fixes a typo. Contributing guide may need an update for new learning center but thinking as long as there's something there and the readme links aren't broken we can fix later.

### CODE REVIEW

**Comment by ch264** [comment]
@SueSmith The markdownlinter failed. 

You can check the errors by clicking on 'Details' and open the dropdown next to markdownlinter in the CI workflow.

**Comment by SueSmith** [comment]
@ch264 yep it's fussy about having four spaces in indented lists etc, fixing..

**Comment by SueSmith** [comment]
@ch264 finally I have appeased it 😅

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+When contributing to this repository, please first discuss the change you wish to make via [Issues](https://github.com/postmanlabs/postman-docs/issues).
+
+Please note we have a [code of conduct](https://community.getpostman.com/t/postman-code-of-conduct/5), please follow it in all your interactions with the project.
+
+## Development Process
+
+We follow [Git Flow](https://guides.github.com/introduction/flow/) for changes to this repository.
+
+  1. Fork this repo to create a copy hosted on your github account. The Fork button is in the top right of the page.
```
Comment (kaustavdm) [inline CONTRIBUTING.md]: @SueSmith This list should be an unordered list.

Comment (SueSmith) [reply CONTRIBUTING.md]: this is literally just the previous file copied over, just trying to get something in the repo so readme link isn't broken but will likely need a redraft at some point, fixed for now anyway

**Comment by kaustavdm** [comment]
Thanks @SueSmith!

**Pull request merged**